sources/packages/web/src/constants/routes/RouteConstants.ts

```diff
@@ -77,6 +77,9 @@ export const InstitutionRoutesConst = {
   STUDENT_FILE_UPLOADS: Symbol(),
   STUDENT_OVERAWARDS: Symbol(),
   STUDENT_NOTES: Symbol(),
+  ASSESSMENTS_SUMMARY: Symbol(),
+  APPLICATION_EXCEPTION: Symbol(),
+  STUDENT_APPEAL_REQUEST: Symbol(),
```

Comment:
Same route is present in student routes const as STUDENT_APPEAL_REQUESTS. One or the other way we can follow the same in both the places.

Reply:
I didn't understand the comment, Are you saying to share the route constants? we never shared the route constants